components/mpas-ocean/bld/namelist_files/namelist_definition_mpaso.xml

```diff
@@ -1196,7 +1196,7 @@ Default: Defined in namelist_defaults.xml
 
 <entry id="config_sw_absorption_type" type="char*1024"
 	category="shortwaveRadiation" group="shortwaveRadiation">
-Name of shortwave absorption type used in simulation. 
+Name of shortwave absorption type used in simulation.
```

Comment:
This may have to be fixed in Registry instead of by hand in this generated file

Reply:
I fixed this in the registry as well.

Reply:
I use an editor that automatically trims trailing whitespace (which I wish more folks would do).  It seems easier to fix this than to revert it.